### DISCUSSION TITLE
Remove support-v4 shadow

### DISCUSSION THREAD

**Discussion by utzcoz** [robolectric/robolectric#7005] [Ideas]
We have deprecated shadow for support-v4(https://github.com/robolectric/robolectric/pull/6194), and  planed to remove it on 4.8 release. Is it time to remove support-v4 shadow now?

Note: If we decide to remove it at current release cycle, we can create an issue from this discussion.

**Comment by hoisie** [comment]
Yeah let's remove it :) Feel free to send a PR to remove it.

**Reply by utzcoz** [reply]
Let's do it. I will send a PR for reviewing this weekend.

**Reply by utzcoz** [reply]
Deprecated classes were removed, and will be released at 4.8. Left classes are marked deprecated again, and planned to be removed at 4.9.

**Reply by utzcoz** [reply]
Related PR: https://github.com/robolectric/robolectric/pull/7031.

**Reply by utzcoz** [reply]
Another PR: https://github.com/robolectric/robolectric/pull/7092.

**Reply by utzcoz** [reply]
Latest PR https://github.com/robolectric/robolectric/pull/7339 to remove support-v4 left things.

**Reply by utzcoz** [reply]
The PR to add removing info for robolectric.github.io is here: https://github.com/robolectric/robolectric.github.io/pull/116.
